.github/cases/blastoise/default.json

```diff
@@ -498,5 +498,16 @@
   "codegen.vfredosum_vs": 142145,
   "codegen.vfredusum_vs": 99859,
   "codegen.vfredmax_vs": 99859,
-  "codegen.vfredmin_vs": 99859
+  "codegen.vfredmin_vs": 99859,
+  "rvv_bench.ascii_to_utf16": 1541742,
+  "rvv_bench.ascii_to_utf32": 673956,
+  "rvv_bench.byteswap": 3456435,
+  "rvv_bench.chacha20": 0,
```

Comment:
strange, it seems chacha20 and poly1305 doesn't goes to the auto-vectorization path. We may need to bump clang or switch to gcc?